hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -106,8 +106,7 @@ public void handle(@NonNull final HandleContext handleContext) {
 
         final var topicUpdate = handleContext.body().consensusUpdateTopic();
         final var topicStore = handleContext.writableStore(WritableTopicStore.class);
-        final var maybeTopic = requireNonNull(topicStore)
-                .get(topicUpdate.topicIDOrElse(TopicID.DEFAULT).topicNum());
+        final var maybeTopic = requireNonNull(topicStore).get(topicUpdate.topicIDOrElse(TopicID.DEFAULT));
```

Comment:
Probably not this PR, but we definitely should not be using `requireNonNull` for topicStore here.  We should be checking if topicStore is null and throwing an appropriate HandleException if it's not present. That allows the workflow to appropriately handle the error, rather than sending an UNKNOWN response and possibly failing the node (currently NPE ends up in catch-all, which also handles `Error`s).

Reply:
We need to change it all over the place because it also exist in the file service.
Let's create a separate ticket for it